### PR TITLE
fix D-PAD nav on API <=25

### PR DESCRIPTION
**Pull request by tsynik**
Described in #5886

### CODE REVIEW

**Comment by vvb2060** [comment]
I don't know why so many modifications are needed #5890

**Comment by tsynik** [comment]
> don't know why so many modifications are needed

some of them are Android Studio xml format, for log window there is initial focus on log frame for able o scroll, for superuser log windows there is focus attribute to able to scroll and o access Vack arrow button. For Modules there is clickable aand focusable options to allow select full block not only eelements on right with action to on/off module (bu you can just off) and focusRight to switches

**Comment by vvb2060** [comment]
For log, it should be consistent with other tabs and not move the focus.

**Comment by vvb2060** [comment]
For other, keep consistent with existing API > 26 behavior

**Comment by tsynik** [comment]
> For log, it should be consistent with other tabs and not move the focus.

Where is initial focus when you click Log in main toolbar? It switches to Button and toolbar, not to actual viewed log on API 25, but if you click floating button and go to Superuser log and Back - Magisk log will be scrollable. You can't scroll it initialy.

**Comment by tsynik** [comment]
> or other, keep consistent with existing API > 26 behavior

This behavior doesn't changed at all, all elements still do what they did. But in other parts of App you can navigate other full blocks, but not in modules. The same for DenyList and that's why you can't access process switches (full block not clickable so no expanded). Add full block on Modules clickable and focuable make it consistent with others.

**Comment by vvb2060** [comment]
keep sync with https://github.com/topjohnwu/Magisk/issues/5834#issuecomment-1140289537

**Comment by tsynik** [comment]
> For log, it should be consistent with other tabs and not move the focus.

here is video where you can see user unable to return from 2nd log window at all without mouse - https://user-images.githubusercontent.com/1551446/171092370-54b667eb-939a-4243-b1e4-00ca05289077.mov

**Comment by vvb2060** [comment]
tv nav controller

**Comment by tsynik** [comment]
> You just need to tap the back button

I pressed Back button on remote to return from Superuser log to Magisk log an now I can't navigate at all on real device with AndroidTV 11 (API 31). So this suggestion do Back but breaks navigation at all.

**Comment by tsynik** [comment]
> For other, keep consistent with existing API > 26 behavior

In earlier Magisk releases Modules module block was selectable and clickable and it pops full module description onClick and highlighted on select as I remember.

File: app/src/main/res/layout/item_module_md2.xml [hunk 1]
```diff
@@ -26,9 +26,12 @@
         <com.google.android.material.card.MaterialCardView
             style="@style/WidgetFoundation.Card"
             isEnabled="@{!item.removed &amp;&amp; item.enabled &amp;&amp; !item.showNotice}"
+            android:clickable="true"
+            android:focusable="true"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:nextFocusRight="@id/module_indicator"
```
Comment (tsynik) [inline app/src/main/res/layout/item_module_md2.xml:33]: Note here is still nextFocusRight

**Comment by yujincheng08** [comment]
close because #5890 has merged

**Comment by tsynik** [comment]
> close because #5890 has merged

Log navigation issue still not fixed as modules inconsisency

**Pull request closed without merging**